package Exception;
import Exception.Log.login1;

public class DZ {

    public static void main(String[] args) {
        new login1();

    }
    
}
